priv/templates/_admin_edit_content_page_connections.tpl

```diff
@@ -28,6 +28,48 @@
                     tabs_enabled=["new"]
                 %}
 
+                <hr>
+            {% elseif name == 'hasextra_img' %}
+                <h4>{{ p.title }}</h4>
+
+                {% live template="_admin_edit_content_page_connections_list.tpl"
+                    topic={object id=id predicate=name}
+                    id=id
+                    predicate=name|as_atom
+                    button_label=button_label
+                    button_class=button_class
+                    dialog_title_add=dialog_title_add
+                    callback=callback
+                    action=action
+                    nocatselect=nocatselect
+                    content_group=content_group
+                    unlink_action=unlink_action
+                    undo_message_id="unlink-undo-message"
+                    list_id=list_id
+                    tabs_enabled=["find", "new"]
```

Comment:
Have you considered also enabling the `upload` tab here? 🤔 

Moreover, if so, this branch of the `if` could also be merged with the one for `hasextra_doc`.